code/datums/mind.dm

```diff
@@ -1384,11 +1441,9 @@
 
 /datum/mind/proc/make_Traitor()
 	if(!(src in SSticker.mode.traitors))

```

Comment:
Perhaps you should check for a pre-existing antag datum first?  It seems more reliable incase adminfuckery goes wrong.